api/content-script/listeners/settings-updated.ts

```diff
@@ -13,13 +36,19 @@ export const watchSettingsUpdated = () => {
         window.location.host,
       );
 
+      window.postMessage(
```

Comment:
Are we not running the risk of sending the `kas:host_connected` event twice from the same routine? (With the other one from above L19)

Reply:
You are right, I didn't notice it. We only need the L19 one, thanks!

Reply:
https://github.com/forbole/kastle/pull/114/commits/c088a727840ab7909efcd113b044e6fd0a997d52
Remove redundant one.